doc-botonic/docs/concepts/14.humanhandoff.md

```diff
@@ -202,3 +200,74 @@ export default class extends React.Component {
 ```
 
 **Note:** Remember that the full features for `getOpenQueues` and `humanHandOff` are only available when the bot is deployed!
+
+## Available Options
+
+Here is the list of the different handoff options available.
+
+| Option             | Description                                                                                                                                          | Parameter     | Type    | Req. |
+|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|---------|------|
+| `withQueue`        | ID or name of the queue where the conversation is transferred.                                                      | queueNameOrId | string  | Yes  |
+| `withOnFinishPath` | Specify a payload or a path to trigger the action that the bot will follow after the conversation with the agent has finished. | path          | string  | Yes  |
+| `withAgentEmail`   | Agent's email.                                                                                                      | email         | string  | Yes  |
+| `withAgentId`      | Agent's ID.                                                                                                         | agentId       | string  | Yes  |
+| `withNote`         | Leave a message for other agents to know more about the case.                                                                  | note          | string  | Yes  |
+| `withCaseInfo`     | Give details about the case.                                                                                                   | caseInfo      | string  | Yes  |
+| `withShadowing`    | Hidie agent's ID and name.                                                                                                  | shadowing     | boolean | No   |
```

Comment:
none of these are strictly required, in fact if you just do a simple handoff it will be done to the first queue (the first one sorted by id) from the organization. No need to put required: Yes, but it's important to add this brief explanation just below the table.